onboarding/src/Components/FormsEdit/FormsEditPage.js

```diff
@@ -51,13 +51,29 @@ function FormsEditPage() {
     }
   }, [saved]);
 
+  const updateUnsetAsNew = function(newSections){
+    if(typeof newSections === 'undefined' || newSections === null)
```

Comment:
I suppose 'undefined' should be without quotation marks?
```
undefined === undefined //true
undefined === 'undefined' //false
```

Reply:
It was the source of my decision :-)
[StackOverflow about undefined - stackoverflow.com/questions/10895288/undefined-typeof-undefined-hasownproperty](https://stackoverflow.com/questions/10895288/undefined-typeof-undefined-hasownproperty)
and there is "`a.c === undefined` will produce an unexpected result if someone has done `undefined = 'defined'` or some such trick".

Nothing else has crossed my mind.
The purpose of this is to leave the function in `if`s cases - is there anything better?